src/SearchResults.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import CustomerProfile from "./CustomerProfile";
+
+const SearchResults = item => {
+  const [isSelected, setIsSelected] = useState(true);
+  const [clickIdProfile, setClickIdProfile] = useState(undefined);
```

Comment:
You can just leave the `useState` empty here, without actually sertting the default to `undefined`. But its not an issue